src/stars.css

```diff
@@ -0,0 +1,91 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  margin-bottom: 4px;
+}
+
+/* .stars__star:last-child {
+  margin-right: 0;
+} */
+
+.stars__star:nth-last-child(-n+5) {
+  margin-bottom: 0;
+}
+
+.stars--0 {
+  .stars__star:nth-child(n) {
+    background-image: url(images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+}
+
+.stars--1 {
+  .stars__star:nth-child(n) {
+    background-image: url(images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+  .stars__star:first-child {
+    background-image: url(images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+}
+
+.stars--2 {
+  .stars__star:nth-child(n) {
+    background-image: url(images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+  .stars__star:nth-child(-n+2) {
+    background-image: url(images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+}
+
+.stars--3 {
+  .stars__star:nth-child(n) {
+    background-image: url(images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+  .stars__star:nth-child(-n+3) {
+    background-image: url(images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+}
+
+.stars--4 {
+  .stars__star:nth-child(n) {
+    background-image: url(images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+  .stars__star:nth-child(-n+4) {
+    background-image: url(images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+}
+
+.stars--5 {
+  .stars__star:nth-child(n) {
+    background-image: url(images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+  .stars__star:nth-child(-n+5) {
+    background-image: url(images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
```

Comment:
don't duplicate code. 
You can move the common styles to `stars__star` and set proper images for needed lines using selectors like this
`.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2),
...
`

Reply:
And I have another doubt here. Doing this, the line will be more than 80 charachters. How can I style this code in this case?